README.md

```diff
@@ -118,7 +118,7 @@ then replace the pipeline in the above command by `pipeline:=pilz_command_planne
 If you've created an application package with your own launch file as described in the
 [tutorials](https://wiki.ros.org/pilz_robots/Tutorials/ModelYourApplicationWithPRBT#Create_your_application_ROS_package),
 you can easily adjust many other configuration parameters.
-See the template and comments in the [pilz_tutorials package](https://github.com/PilzDE/pilz_tutorials).
```

Comment:
is this the right place for it?

Reply:
Yes! Because pilz_templates is an essential package, which is used by nearly every customer working with our robot. Because this readme is the main page and entry point, I would mention it there to avoid confusion.